Clinical-Genomics/scout
# Use a config file at the time when launching the cli, the web app or the demo version

Demo config file is read and used also on clinical-db stage and prod.
This is an attempt at making the app use a unique config file, either one provided by the command line, or the demo one (last option)

**How to test**:
1. Test it locally and it should debug params from the demo config file
2. Install it on clinical-db stage, along with Clinical-Genomics/servers#598 (fixed scout-demo config file)
2. Scout web should work as usual
3. Scout cli should work as usual (run `scout view cases` command for instance)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
